docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -43,19 +41,18 @@ To complete this tutorial, you'll need:
 * An {es} cluster with hot and warm data tiers.
 
 ** {ess}:
-Elastic Stack deployments on {ess} include a hot tier by default. To add a warm
-tier, edit your deployment and click **Add capacity** for the warm data tier.
-+
+Elastic Stack deployments on {ess} automatically include hot and warm tiers. If needed, you can add additional warm capacity by editing your deployment and selecting **Add capacity** for the warm data tier.
```

Comment:
They do not automatically include a `warm` tier. They include a `hot` and `content` tier.
```suggestion
Elastic Stack deployments on {ess} automatically include hot and content tiers. If needed, you can add additional warm capacity by editing your deployment and selecting **Add capacity** for the warm data tier.
```